src/components/screens/vacation-requests/vacation-requests-table/vacation-requests-table-toolbar/toolbar-title.tsx

```diff
@@ -0,0 +1,27 @@
+import { useEffect, useState } from "react";
+import { ToolbarFormModes } from "../../../../../types";
+import { Typography } from "@mui/material";
+import { getToolbarTitle } from "../../../../../utils/toolbar-utils";
+
+/**
+ * Component properties
+ */
+interface Props {
+  toolbarFormMode: ToolbarFormModes;
+}
+
+/**
+ * Toolbar title component
+ */
+const ToolbarTitle = ({ toolbarFormMode }: Props) => {
+  const [title, setTitle] = useState<string>("My vacation requests");
```

Comment:
No need to state the type for the useState as it can be inferred from the initial value

Reply:
Should this initial value be loocalized?

Reply:
Yeah. Inferred type and localized initial value.